service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json

```diff
@@ -0,0 +1,761 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "clusterLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Location of the Cluster"
+      }
+    },
+    "clusterName": {
```

Comment:
I think the probabilities of failing are the same… any how,, can you send me the exact Json (snippet) on what you are talking about and how we can validate in JSON that a certain VIP will be unique.

Also – while we try and resolve these differences, I would recommend that you merge what we have got now to main… Folks have been asking about them And I have now started to point them to my on Git repo.. and are using them without issues... I do not want to make that the main place folks go to for SF templates..

From: Kay Singh [mailto:notifications@github.com]
Sent: Friday, December 4, 2015 9:31 AM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Chacko Daniel <chackdan@microsoft.com>
Subject: Re: [azure-quickstart-templates] WAD 5 Node (#863)


In service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/863#discussion_r46707965>:

> @@ -0,0 +1,761 @@

> +{

> +  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",

> +  "contentVersion": "1.0.0.0",

> +  "parameters": {

> +    "clusterLocation": {

> +      "type": "string",

> +      "metadata": {

> +        "description": "Location of the Cluster"

> +      }

> +    },

> +    "clusterName": {

@ChackDan<https://github.com/ChackDan> But it could be the difference between the user entering the value "testcluster" or "uniqueString>ClusterName". The first will lead to deployment failures and user will need to start again.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/863/files#r46707965>.